core/logger/Logger.cpp

```diff
@@ -267,14 +286,32 @@ void Logger::LoadConfig(const std::string& filePath) {
         logConfigInfo = "Load log config from " + filePath;
     } while (0);
 
+    // parse env log level
+    level::level_enum* envLogLevel = new(level::level_enum);
```

Comment:
using level::level_enum envLogLevel to define a var is enough

Reply:
I think using a pointer here is more suitable to decide the env value parsed correctly,  the default value is trace?  Pls help me to do a better choice.

Reply:
I think it can be INFO. And its value is only valid only when aliyun_logtail_log_level is not empty